### PR TITLE
[WIP - DO NOT MERGE] Handle opening project from microbit.org

### PR DESCRIPTION
**Pull request by microbit-grace**
- [ ] Remove debug logging

### CODE REVIEW

File: lang/ui.en.json [hunk 1]
```diff
@@ -71,6 +71,10 @@
     "defaultMessage": "Close",
     "description": "Close button text or label"
   },
+  "code-download-error": {
+    "defaultMessage": "Error downloading the target code",
+    "description": "Title for error message relating to project loading"
+  },
```
Comment (microbit-grace) [inline lang/ui.en.json:77]: One string added. Do not merge till strings clean up is merged

File: src/pages/ImportPage.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { useStore } from "../store";
+import { createSessionPageUrl } from "../urls";
+
+const ImportPage = () => {
+  const navigate = useNavigate();
+  const intl = useIntl();
+  const { activitiesBaseUrl } = useDeployment();
+  const resource = useMicrobitResourceSearchParams();
+  const loadProject = useStore((s) => s.loadProject);
+  console.log("resource", resource);
```
Comment (microbit-grace) [inline src/pages/ImportPage.tsx:18]: To remove

File: src/pages/ImportPage.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    const updateAsync = async () => {
+      if (!resource || !activitiesBaseUrl) {
+        return;
+      }
+      const code = await fetchMicrobitOrgResourceTargetCode(
+        activitiesBaseUrl,
+        resource,
+        intl
+      );
+      console.log("code", code);
```
Comment (microbit-grace) [inline src/pages/ImportPage.tsx:30]: To remove

File: src/pages/HomePage.tsx [hunk 4]
```diff
@@ -63,13 +68,88 @@ const graphData = {
...
+    const updateAsync = async () => {
+      if (!resource || !activitiesBaseUrl) {
+        return;
+      }
+      const code = await fetchMicrobitOrgResourceTargetCode(
+        activitiesBaseUrl,
+        resource,
+        intl
+      );
+      console.log("code", code);
```
Comment (microbit-grace) [inline src/pages/HomePage.tsx:146]: To remove

File: src/pages/HomePage.tsx
Comment (microbit-grace) [inline src/pages/HomePage.tsx:1]: Revert changes in HomePage

File: lang/ui.en.json (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply lang/ui.en.json:93]: I've now merged that

**Comment by microbit-grace** [comment]
Renamed branch
Continued in https://github.com/microbit-foundation/ml-trainer/pull/388

**Pull request closed without merging**